mop/brokers/test_asassn.py

```diff
@@ -0,0 +1,70 @@
+#import mytom.harvest_asassn
+from mytom.samplecode import ASASSNBroker
+import unittest
+import requests
+from unittest import mock
+from unittest.mock import Mock
+from django.test import TestCase, override_settings, tag
+from unittest.mock import patch
+import lxml.html as lh
+from tom_targets.models import Target
+from tom_alerts.alerts import GenericBroker, GenericQueryForm
+from tom_dataproducts.models import ReducedDatum
+
+BROKER_URL = 'http://www.astronomy.ohio-state.edu/asassn/transients.html'
+broker = ASASSNBroker('ASAS-SN broker')
+fakedata = [['id',['','']],['other',['AT2021kdo (= Gaia21bxn)',
+'AT20210du(=Gaia21cqi)']],['ATEL',['','']],['RA',['1:6:42.74','8:8:36.48']],
+['Dec',['61:59:40.9','-40:53:23.5']],['Discovery',['2021-06-9.44','2021-06-12.74']],
+['V/g',['13.98','15.46']],['SDSS',['','']],['DSS',['','']],['Vizier',['','']],
+['Spectroscopic Class',['','']],
+['comments',['known microlensing event or Be-type outburst, discovered 2021/04/16.801','known candidate Be-star or microlensing event, discovered 2021/06/01.188']]]
+
+
+class TestActivity(unittest.TestCase):
+    def SetUp(self,broker):
+        self.broker = ASASSNBroker('ASAS-SN Broker')
+        broker = ASASSNBroker('ASAS-SN broker')
+        
+        #harvestasassn = HarvestAsassn()
+    '''    
```

Comment:
Several methods here are actually part of the doc string and so are not in use.  If these lines are intended as documentation, it should be clarified and in English phrases or code examples, otherwise they need to be integrated with the code.